### DISCUSSION TITLE
Use native ffmpeg bindings for even more speed

### DISCUSSION THREAD

**Discussion by newadventure079** [theophanemayaud/video-simili-duplicate-cleaner#133] [Ideas]
This repo https://github.com/0x90d/videoduplicatefinder uses native ffmpeg bindings and it really does speed up the process significantly.  Maybe this project could do the same thing too.

**Comment by theophanemayaud** [comment]
Hey ! Sorry for the delay 😅
Actually Video simili duplicate cleaner already uses «native bindings», meaning it uses ffmpeg as a library, not a standalone executable. When switching over to that approach #13 there was indeed a significant boost !
